.github/workflows/rust.yml

```diff
@@ -2,9 +2,17 @@ name: Rust
 
 on:
   push:
-    branches: ["main"]
+    branches: [ "main" ]
+    paths:
+      - '**.rs'
+      - '**.toml'
+      - '**.json'
   pull_request:
-    branches: ["main"]
+    branches: [ "main" ]
+    paths:
+      - '**.rs'
+      - '**.toml'
+      - '**.json'
```

Comment:
Is there any chance of missing important files down the line with this change? If so, maybe we can exclude paths instead (`*.md`) if Github Actions allows it

Reply:
Good point, testing